packages/worker/scripts/dev/manage.js

```diff
@@ -29,6 +29,7 @@ async function init() {
       SERVICE: "worker-service",
       DEPLOYMENT_ENVIRONMENT: "development",
       TENANT_FEATURE_FLAGS: "*:LICENSING,*:USER_GROUPS,*:ONBOARDING_TOUR",
+      ENABLE_EMAIL_TEST_MODE: 1,
```

Comment:
Should this be on in all dev environments?

Reply:
Should be good, all the email sending is nicely encapsulated in the worker. I'll add something similar to the account portal when I move onto the sso updates there too